components/builder-api/doc/api.raml

```diff
@@ -214,6 +242,54 @@ securitySchemes:
                                 }
                 400:
                     description: Received a jobId that was not a number
+        /log:
+            get:
+                description: |
+                  Get the build log (or a portion thereof) for the
+                  given job. Until an SSE endpoint can be implemented,
+                  this endpoint can be used to poll for the logs of a
+                  currently-running job; use of the `start` query
+                  parameter ensures that only the portion of the log
+                  that is "new", relative to a prior request, will be
+                  retrieved.
+                queryParameters:
+                    start:
+                        description: |
+                            Return the portion of the log output that
+                            starts at the zero-indexed line
+                            specified. A request with no given value
+                            defaults to `0`; in other words, the
+                            entire log will be sent back.
+                        type: integer
+                        required: false
+                        default: 0
+                        minimum: 0
+                        example: 100
+                responses:
+                    200:
+                        body:
+                            application/json:
+                                schema: jobLog
+                                example: |
+                                    {
+                                        "start": 0,
+                                        "stop": 2,
+                                        "content": [
+                                            "this is the first line",
```

Comment:
Right now, the API is actually returning `"1: this is the first line"` (i.e., the string includes the line number). Can we remove that, and just return the content?